tools/run_desktop_tests.py

```diff
@@ -156,6 +164,9 @@ def run_tests(self, tests_to_run):
 
 
     def log_exec_file(self, filename, result=None):
+        if self.boost_tests:
```

Comment:
As is said in the introduction comment to this pr, the resulting XML will be processed by Jenkins directly. Our home-grown test library gives out non-standard output, so we need to turn it into something Jenkins can understand, with boost tests we don't.